easybuild/tools/options.py

```diff
@@ -720,10 +720,6 @@ def postprocess(self):
             build_easyconfig_constants_dict()  # runs the easyconfig constants sanity check
             self._postprocess_list_avail()
 
-        # if --backup-modules is used without --module-only print a warning
```

Comment:
Mmmh, I'd like to keep the warning. Otherwise you might end up without a backup, even though you have that option listed, and you don't know why.